build.xml

```diff
@@ -667,11 +669,29 @@ Copyright (c) 2005, 2015, Oracle and/or its affiliates. All rights reserved.
             </not>
         </condition>
     </target>
+    
+    <target name="check_plugins_jar" description="check if plugins.jar was generated">
+        <available
+            file="${build.test.classes.dir}/org/opensolaris/opengrok/authorization/plugins.jar"
+            type="file" property="plugins_exists">
+        </available>
+        <condition property="plugins_missing">
+            <not>
+                <isset property="plugins_exists" />
+            </not>
+        </condition>
+    </target>
 
-    <target depends="create_repos" name="-post-compile-test">
+    <target depends="create_repos, create_plugins" name="-post-compile-test">
```

Comment:
fwiw - the mvn build is used byt Travis ... (hence the motivation to keep it up to date ... we should probably move just to mvn, but we still have lots of unmigrated code in ant, e.g. release cycle, etc.)